README.md

```diff
@@ -27,38 +32,61 @@ The CI performs the following steps:
-
-The Linux images for the `linux_litex` example can be build following the [linux on litex vexriscv](https://github.com/litex-hub/linux-on-litex-vexriscv) instructions.
-
-The Travis-based CI performs all the necessary steps to build the example designs and generate the bitstreams.
-If you want to manually build the examples, run following commands:
+* Conda set up
 
 ```bash
-git clone https://github.com/SymbiFlow/symbiflow-examples.git
-cd symbiflow-examples
-wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-bash miniconda.sh -b -p $HOME/miniconda
-source "$HOME/miniconda/etc/profile.d/conda.sh"
-conda config --set always_yes yes --set changeps1 no
+mkdir env
+cd env
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh -b -p `pwd`/conda
+source `pwd`/conda/etc/profile.d/conda.sh
+conda config --system --add envs_dirs conda/envs
+conda config --system --add pkgs_dirs conda/pkgs
+conda config --system --set always_yes yes
+conda config --system --set changeps1 no
 conda config --add channels conda-forge
 conda config --add channels symbiflow
 conda update -q conda
 conda activate
+```
+
+* Conda package install
+
+```bash
 conda install -c symbiflow yosys
 conda install -c symbiflow yosys-plugins
 conda install -c symbiflow vtr=8.0.0.rc2_3575_g253f75b6d
 conda install lxml simplejson intervaltree python-constraint git pip
 pip install git+https://github.com/symbiflow/fasm
-wget "https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/4/20200416-002215/symbiflow-arch-defs-install-a321d9d9.tar.xz"
-tar -xf symbiflow-arch-defs-install-a321d9d9.tar.xz
-export PATH=$PWD/install/bin:$PATH
+```
+
+* Symbiflow toolchain download
+
+```bash
+wget -O symbiflow.tar.xz https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/4/20200416-002215/symbiflow-arch-defs-install-a321d9d9.tar.xz
+tar -xf symbiflow.tar.xz
+rm symbiflow.tar.xz
+```
+
+## Build Example Designs
+
+The example designs are provided in separate directories:
+
+1. `counter` - simple 4-bit counter driving LEDs. The design targets the [Basys3 board](https://store.digilentinc.com/basys-3-artix-7-fpga-trainer-board-recommended-for-introductory-users/) .
+2. `picosoc` - [picorv32](https://github.com/cliffordwolf/picorv32) based SoC. The design targets the [Basys3 board](https://store.digilentinc.com/basys-3-artix-7-fpga-trainer-board-recommended-for-introductory-users/).
```

Comment:
you don't really have to manually set list item numbers https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists